docs/LV_client/tools/mono_collision_plots.md

```diff
@@ -0,0 +1,29 @@
+# mono_collision_plots.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_collision_plots.png"/>
+</p>
+
+### Description
+
+Tool to plot the information provided by the collision sensor.
+
+For technical support contact us at <b>support@monodrive.io</b> 
+ 
+
+### Inputs
+
+- **Collision sensor sample:**  The accumulatative samples of the Collision sensor
+ 
+
+- **error in (Error Cluster):** can accept error information wired from VIs previously called. Use this information to decide if any functionality should be bypassed in the event of errors from other VIs. 
+
+### Outputs
+
+- **Collision Ocurred:**  True if a collision if the ego collides with other object on
+the scene
```

Comment:
True if there is a collision or if the ego collides with another object on
the scene